frontend/src/Settings/Notifications/Notifications/AddNotificationModalContent.js

```diff
@@ -33,32 +34,32 @@ class AddNotificationModalContent extends Component {
         <ModalBody>
           {
             isSchemaFetching &&
-              <LoadingIndicator />
+            <LoadingIndicator />
```

Comment:
Since these are being touched for some reason, convert to ternary instead..

Reply:
Also, Indent was correct on these before, don't think we need any changes to this file